src/CMakeLists.txt

```diff
@@ -18,6 +18,54 @@ mark_as_advanced(
 )
 add_subdirectory(crc32c EXCLUDE_FROM_ALL)
 
+set(LEVELDB_BUILD_TESTS OFF CACHE BOOL "" FORCE)
+set(LEVELDB_BUILD_BENCHMARKS OFF CACHE BOOL "" FORCE)
+set(LEVELDB_INSTALL OFF CACHE BOOL "" FORCE)
+set(HAVE_CRC32C ON CACHE BOOL "" FORCE)
+set(HAVE_SNAPPY OFF CACHE BOOL "" FORCE)
+set(HAVE_TCMALLOC OFF CACHE BOOL "" FORCE)
+# Disabling TSA warnings because leveldb simultaneously
```

Comment:
I don't understand this comment? Is there a TSA warning that turns into an error?

Reply:
This code is [dropped](https://github.com/hebasto/bitcoin/pull/6#issuecomment-1435777703) now.